man/kernel-install.xml

```diff
@@ -211,7 +211,9 @@
           <filename>/etc/machine-id</filename>
         </term>
           <listitem>
-            <para>The content of the file specifies the machine identification <replaceable>MACHINE-ID</replaceable>.</para>
+            <para>The content of the file specifies the machine identification
+            <replaceable>MACHINE-ID</replaceable>. If it cannot read /etc/machine-id, kernel-install will use
```

Comment:
```suggestion
            <replaceable>MACHINE-ID</replaceable>. If <filename>/etc/machine-id</filename> cannot be read, kernel-install will use
```
Although I am not super sure, whether all instances of filenames need to be enclosed in those tags.

Reply:
Fixed to use <filename> but I think we should stick to active case instead of passive case.